general-configuration-guide/basic-setup.md

```diff
@@ -52,7 +51,7 @@ tinymce.init({
 
 {{site.productname}} can also match an `id` attribute.
 
-The following example replaces a `textarea` element with the `id` `'default'` on the page:
+The following example replaces a `textarea` element with the `id="default"` on the page:
```

Comment:
This reads really weirdly? I think the older version was clearer, but maybe I'm just misreading.

Reply:
reverted